app/src/components/ProposalItem.tsx

```diff
@@ -38,11 +38,14 @@ class ProposalItem extends React.Component<ProposalItemProps, ProposalItemState>
       .call()
       .then((proposalData: any): void => {
         const proposal = {
-          title: proposalData[2],
-          medium: proposalData[3],
-          edition: proposalData[4],
-          created: proposalData[5],
-          metaUri: proposalData[6],
+          title: proposalData[1],
```

Comment:
In Proposal Item, we call ArtifactApplication to get the proposal, in this call we also get back who called the requested the proposal in the first position of the tuple, in which case we actually need to offset all these indexes by 1